Crashlytics/Crashlytics/Controllers/FIRCLSRolloutsPersistenceManager.m

```diff
@@ -57,7 +64,11 @@ - (void)updateRolloutsStateToPersistenceWithRollouts:(NSData *_Nonnull)rollouts
 
   NSFileHandle *rolloutsFile = [NSFileHandle fileHandleForUpdatingAtPath:rolloutsPath];
 
-  dispatch_async(FIRCLSGetLoggingQueue(), ^{
+  if (!_rolloutsLoggingQueue) {
+    FIRCLSDebugLog(@"Rollouts logging queue is dealloccated");
+  }
```

Comment:
I think you are right, but that's also my question for why previous `FIRCLSGetLoggingQueue()` getting deallocated. The default logging queue have attribute declaration under [FIRCLSGlobals](https://github.com/firebase/firebase-ios-sdk/blob/main/Crashlytics/Crashlytics/Components/FIRCLSGlobals.h) (should default a strong reference?) and gets initialized under [FIRCLSContext](https://github.com/firebase/firebase-ios-sdk/blob/main/Crashlytics/Crashlytics/Components/FIRCLSContext.m#L240). Everything is under the process of initialize [report manager](https://github.com/firebase/firebase-ios-sdk/blob/main/Crashlytics/Crashlytics/FIRCrashlytics.m#L200) which is the last process before init Rollouts related stuff. And this init method `[_reportManager startWithProfiling]` returns a Promise so I assume the operation is in serial order. For me feel like `FIRCLSGetLoggingQueue()` could never being deallocated during the app run. But I could miss something without actual reproduce the issue since the logic over there is so complex :)

Reply:
Hmm, yeah I'm confused too. Something weird has to happen for original crash to happen. This is a good fix and the extra logging could hopefully help debug future issues, if they arise.